src/components/ToMarkdown/README.md

```diff
@@ -0,0 +1,19 @@
+# To Markdown
+
+**Renders nested HTML content to Markdown.**
+
+## Examples
```

Comment:
I wanted to write these and others as test. But while this component works in a route, I'm not able to isolate the output properly. 🤷 .